packages/admin-ui/src/node/utils/validate-args.ts

```diff
@@ -0,0 +1,28 @@
+import { CustomWebpackConfigArgs } from "../types"
+import { logger } from "./logger"
+
+function validateArgs(args: CustomWebpackConfigArgs) {
+  const { options } = args
+
+  if (options.path) {
+    if (!options.path.startsWith("/")) {
```

Comment:
question: is this breaking? I think we've been using `path: app` in staging so far

Reply:
Also, if there are any other stuff breaking the beta version, it would be great to have those highlighted in the release notes + upgrade guide.